homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -130,7 +130,7 @@ class AlarmControlPanelEntity(Entity):
     _attr_changed_by: str | None = None
     _attr_code_arm_required: bool = True
     _attr_code_format: str | None = None
-    _attr_supported_features: int
```

Comment:
This changes the behavior of our entity model; which is not allowed. Please revert this change.

Reply:
@frenck if I have rolled back, the test failed:

https://github.com/home-assistant/core/runs/4695510080?check_suite_focus=true#step:8:14

 ************* Module homeassistant.components.tuya.alarm_control_panel
Error: homeassistant/components/tuya/alarm_control_panel.py:112:12: E1101: Instance of 'TuyaAlarmEntity' has no '_attr_supported_features' member (no-member)
Error: homeassistant/components/tuya/alarm_control_panel.py:115:12: E1101: Instance of 'TuyaAlarmEntity' has no '_attr_supported_features' member (no-member)
Error: homeassistant/components/tuya/alarm_control_panel.py:118:12: E1101: Instance of 'TuyaAlarmEntity' has no '_attr_supported_features' member (no-member)
Error: Process completed with exit code 2.

Reply:
nvm it seems it works after put them in tuya/alarm_control_panel.py